tests/documents_tests.ts

```diff
@@ -183,6 +184,20 @@ describe('Documents tests', () => {
         expect(response).toHaveProperty('title', title)
       })
 
+      test(`${permission} key: Partial update a document`, async () => {
```

Comment:
`Partially update a document` or `Partial update of a document`

Reply:
```suggestion
      test(`${permission} key: Partial update of a document`, async () => {
```